src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Figure implements AreaCalculator, Drawable {

```

Comment:
```suggestion
public class Figure implements AreaCalculator, Drawable {

```

do we need to have the ability to create instances of `Figure` class?

Reply:
If I correctly understood the conditions of the task, then it is necessary.
The condition says to create methods that return Figures